### PR TITLE
Implement statistics functionality

### PR DESCRIPTION
**Pull request by Space-zee**
https://app.clickup.com/t/860pt2dad

### CODE REVIEW

File: api/package.json [hunk 1]
```diff
@@ -31,10 +31,12 @@
     "@nestjs/config": "2.2.0",
     "@nestjs/core": "9.2.1",
     "@nestjs/platform-express": "9.2.1",
+    "@nestjs/schedule": "^2.2.0",
```
[the remark this answers is not in the document]
Comment (Space-zee) [reply api/package.json:34]: +

File: api/package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:e2e": "jest --config ./test/jest-e2e.json",
     "typeorm": "node --require ts-node/register ./node_modules/typeorm/cli.js",
-    "typeorm:migration:generate": "npm run typeorm -- -d ./src/common/typeorm.ts migration:generate ./src/migration/order-history",
+    "typeorm:migration:generate": "npm run typeorm -- -d ./src/common/typeorm.ts migration:generate ./src/migration/statistics",
```
[the remark this answers is not in the document]
Comment (Space-zee) [reply api/package.json:24]: +

File: api/src/app.module.ts [hunk 3]
```diff
@@ -6,10 +6,13 @@ import { TypeOrmModule } from '@nestjs/typeorm';
 import { KycModule } from './features/kyc/kyc.module';
 import { AssetModule } from './features/assets/asset.module';
 import { TransactionsHistoryModule } from './features/transactionsHistory/transactionsHistory.module';
+import { ScheduleModule } from '@nestjs/schedule';
+import { StatisticModule } from './features/statistic/statistic.module';
 
 @Module({
   imports: [
     ConfigModule.forRoot({ envFilePath: '.env', isGlobal: true }),
+    ScheduleModule.forRoot(),
```
[the remark this answers is not in the document]
Comment (Space-zee) [reply api/src/app.module.ts:15]: +

File: api/src/features/kyc/rpc.client.ts [hunk 4]
```diff
@@ -42,12 +41,12 @@ export class RpcClient implements OnModuleInit {
...
 
   async onModuleInit(): Promise<void> {
-    const kycOracleAddress = await this.wallet.getAddress()
-    console.log('RPC_URL', this.config.get<string>('RPC_URL'))
-    console.log('RPC_ENV', this.config.get<string>('RPC_ENV'))
-    console.log('kycOracleAddress', kycOracleAddress)
-    if (!await this.accessManagerSc.isKycOracle(kycOracleAddress)) {
-      throw "CRITICAL ERROR: PRIVATE_KEY_KYC_STORE_CONTROLLER is invalid. Permission denied. (Fix this before running server)"
+    const kycOracleAddress = await this.wallet.getAddress();
+    console.log('RPC_URL', this.config.get<string>('RPC_URL'));
```
[the remark this answers is not in the document]
Comment (Space-zee) [reply api/src/features/kyc/rpc.client.ts:45]: +

File: api/src/features/statistic/entity/statistic.entity.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import {Entity, Column, PrimaryGeneratedColumn, CreateDateColumn, UpdateDateColumn} from 'typeorm';
+
+@Entity()
+export class StatisticEntity {
+  @PrimaryGeneratedColumn('increment')
+  id: number;
+
+  @Column()
+  user: string;
```
[the remark this answers is not in the document]
Comment (Space-zee) [reply api/src/features/statistic/entity/statistic.entity.ts:9]: +

**Review by ruslanpalagin — APPROVED** [review]

**Pull request merged**